tests/base_scraper/test_base_scraper.py

```diff
@@ -134,6 +134,13 @@ def test_destination(self):
                                             log_level='ERROR')
         self.assertEqual(scraper.target, destination)
 
+        # destination only has filename
+        scraper = mozdownload.DirectScraper(url=test_url,
+                                      destination=filename,
+                                      version=None,
+                                      log_level='ERROR')
```

Comment:
Just a nit here... Please fix the indentation of the lines 139-141, so they are aligned with `url`. Once this is fixed we can get the PR merged. Thanks a lot!